tests/test_runs/test_run_functions.py

```diff
@@ -410,10 +410,13 @@ def test_check_erronous_sklearn_flow_fails(self):
 
         # Invalid parameter values
         clf = LogisticRegression(C="abc", solver="lbfgs")
-        with self.assertRaisesRegex(
-            ValueError,
-            r"Penalty term must be positive; got \(C=u?'abc'\)",  # u? for 2.7/3.4-6 compability
-        ):
+        # The exact error message depends on scikit-learn version.
+        # Because the sklearn-extension module is to be separated,
+        # I will simply relax specifics of the raised Error.
```

Comment:
Would it be possible to test on a more relaxed regex so we can make sure that we test for the right thing?